default/stringtables/en.txt

```diff
@@ -1886,7 +1886,7 @@ OPTIONS_DB_AUTOSAVE_GALAXY_CREATION
 Enable autosave after the galaxy is created before any game play.
 
 OPTIONS_DB_AUTOSAVE_GAME_CLOSE
-Enable autosave when resigning or closing the game.
+Enable autosave when resigning or closing the game. In multiplayer hostless mode it enables autosave when the server closing the game when player disconnected or the server was stopped.
```

Comment:
`In multiplayer hostless mode, enable autosaves when the server closes a game due to player disconnection or when the server is stopped manually.`

Not sure if the "stopped manually" part is accurate, but rephase accordingly...

Reply:
May be use `... is interrupted/terminated.` as it refers to SIGINT, SIGTERM signals?

Reply:
Fixed.